MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -4327,29 +4312,10 @@ private void stpbtvl()
         /// </summary>
         private void strcmp()
         {
+            var string1 = GetParameterString(0);
+            var string2 = GetParameterString(2);
 
-            var string1Pointer = GetParameterPointer(0);
-            var string2Pointer = GetParameterPointer(2);
-
-            var string1 = Module.Memory.GetString(string1Pointer);
-            var string2 = Module.Memory.GetString(string2Pointer);
-
-#if DEBUG
-            _logger.Info(
-                $"Comparing ({string1Pointer}){Encoding.ASCII.GetString(string1)} to ({string2Pointer}){Encoding.ASCII.GetString(string2)}");
-#endif
-
-            for (var i = 0; i < string1.Length; i++)
-            {
-                if (string1[i] == string2[i]) continue;
-
-                //1 < 2 == -1 (0xFFFF)
-                //1 > 2 == 1 (0x0001)
-                Registers.AX = (ushort)(string1[i] < string2[i] ? 0xFFFF : 1);
-                return;
-            }
-
-            Registers.AX = 0;
+            Registers.AX = (ushort)(short)String.Compare(string1, string2);
```

Comment:
`string` vs `String` alias
Unnecessary cast to `(short`) before `(ushort)`. `int` -1 will cast to `ushort` 0xFFFF

![image](https://user-images.githubusercontent.com/1139047/92239218-a20dd580-ee88-11ea-9476-dd9dc2b3e84b.png)